README.md

```diff
@@ -41,7 +41,7 @@ This is an experimental web client for Jellyfin based on Vue.js. We welcome all
 
 ## Requirements
 
-This project uses [Yarn](https://yarnpkg.com/getting-started/install) as a package manager.
+This project uses [npm](https://docs.npmjs.com/getting-started) as a package manager.
```

Comment:
We can probably just change this to mention we require Node 14, since NPM comes standard with it.